Tests/InfomaniakCoreTests/CoreDB/UTTransaction.swift

```diff
@@ -0,0 +1,264 @@
+/*
+ Infomaniak Core - iOS
+ Copyright (C) 2023 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCore
+import InfomaniakCoreDB
+import RealmSwift
+import XCTest
+
+/// Wrapping a `InMemoryFileDB` for realm access
+struct MCKRealmAccessible: RealmAccessible {
```

Comment:
Since the implementation of the transaction API lives in Core-DB, I test it with an in memory context.